### PR TITLE
fixed audio glitches caused by previous audio concatenation method

### PR DESCRIPTION
**Pull request by tonywiththetommy**
follow-up to [#915](https://github.com/elebumm/RedditVideoMakerBot/pull/915)

created a separate branch that only includes this change

REQUIRES FFMPEG.EXE IN MAIN PROJECT FOLDER

[video example](https://www.youtube.com/watch?v=Wb3pkdWjB-4)

### CODE REVIEW

**Comment by Drugsosos** [comment]
@JasonLovesDoggo @tonywiththetommy 
there is FFMPEG in dependencies tho, not a biggie)

you can use that
```
import imageio_ffmpeg

imageio_ffmpeg.get_ffmpeg_exe()
```

**Comment by JasonLovesDoggo** [comment]
> @JasonLovesDoggo @tonywiththetommy 
> there is FFMPEG in dependencies tho, not a biggie)
> 
> you can use that
> ```
> import imageio_ffmpeg
> 
> imageio_ffmpeg.get_ffmpeg_exe()
> ```

Awesome!

File: TTS/engine_wrapper.py [hunk 1]
```diff
@@ -85,42 +88,84 @@ def split_post(self, text: str, idx: int):
...
+                print("newtext was blank because sanitized split text resulted in none")
+                continue
+            else:
+                self.call_tts(f"{idx}-{idy}.part", newtext)
+                with open(f"{self.path}/list.txt", 'w') as f:
+                    for idz in range(0, len(split_text)):
+                        f.write("file " + f"'{idx}-{idz}.part.mp3'" + "\n")
+                    split_files.append(str(f"{self.path}/{idx}-{idy}.part.mp3"))
+                    f.write("file " + f"'silence.mp3'" + "\n")
+                f.close()
```
Comment (Drugsosos) [inline TTS/engine_wrapper.py:114]: there is a context manager, no need in .close()
```suggestion
                
```

**Comment by Drugsosos** [comment]
I don't think the problem is in concat method, I believe it's in split text and inability of TTS (TikTik's at least) to read dots on the end.
If you'll look closely, you can spot it in the video, there are 2 dots:
  * one after `12 pack of milk cartons and put them in one by one.`
  * and another one after `that day everyone did it my way.`

**Comment by JasonLovesDoggo** [comment]
Merge conflicts

**Comment by JasonLovesDoggo** [comment]
See #1177

**Pull request closed without merging**